Quick_Deploy/HuggingFaceTransformers/README.md

```diff
@@ -0,0 +1,263 @@
+of hosting many models, simultaneously. To accomplish this, first ensure you have
+exited the docker container by invoking `Ctrl+C` and waiting for the container to exit.
+
+Next copy the remaining model provided into the model repository:
+```
+cp -r persimmon8b/ model_repository/
+```
+*NOTE*: The combined size of these two models is large. If your current hardware cannot
+support hosting both models simultaneously, consider loading a smaller model, such as
+[opt-125m](https://huggingface.co/facebook/opt-125m), by creating a folder for it
+using the templates provided and copying it into `model_repository`.
+
+Again, launch the server by invoking the `docker run` command from above and wait for confirmation
+that the server has launched successfully.
+
+Query the server making sure to change the host address for each model:
+```json
+curl -X POST localhost:8000/v2/models/falcon7b/infer -d '{"inputs": [{"name":"text_input","datatype":"BYTES","shape":[1],"data":["How can you be"]}]}'
+curl -X POST localhost:8000/v2/models/persimmon8b/infer -d '{"inputs": [{"name":"text_input","datatype":"BYTES","shape":[1],"data":["Where is the nearest"]}]}'
+```
+In our testing, these queries returned the following parsed results:
+```bash
+# falcon7b
+"How can you be sure that you are getting the best deal on your car"
+
+# persimmon8b
+"Where is the nearest starbucks?"
+```
+
+## 'Day Zero' Support
+
+The latest transformer models may not always be supported in the most recent, official
+release of the `transformers` package. In such a case, you should still be able to
+load these 'bleeding edge' models in Triton by building `transformers` from source.
+This can be done by replacing the transformers install directive in the provided
+Dockerfile with:
+```docker
+RUN pip install git+https://github.com/huggingface/transformers.git
+```
+Using this technique you should be able to serve any transformer models supported by
+hugging face with Triton.
+
+
+# Next Steps
+The following sections expand on the base tutorial and provide guidance for future sandboxing.
+
+## Loading Cached Models
+In the previous steps, we downloaded the falcon-7b model from hugging face when we
+launched the Triton server. We can avoid this lengthy download process  in subsequent runs
+by loading cached models into Triton. To do so, we can follow the hugging face [tutorial
+for downloading models](https://huggingface.co/docs/hub/models-downloading). Once the model
+is downloaded, we can mount it to the Triton container by adding the following mount option to our
+`docker run` command from earlier (making sure to replace {USER} with your username on
+Linux):
+
+```bash
+# Option to mount a specific cached model (falcon-7b in this case)
+-v /home/{USER}/.cache/huggingface/hub/models--tiiuae--falcon-7b:/root/.cache/huggingface/hub/models--tiiuae--falcon-7b
+
+# Option to mount all cached models on the host system
+-v /home/{USER}/.cache/huggingface:/root/.cache/huggingface
```

Comment:
This is a great addition! Do you know if the cache directory is configurable for HF? Would be cool if we could eventually set it to cache where the model.py lives / in the model repository or something.

Reply:
Indeed it is, and indeed we can. I'll extend this section.